### PR TITLE
libnabo: add version 1.1.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **libnabo/1.1.0**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libnabo/all/patches/1.1.0-0001-fix-cmake.patch
Comment (uilianries) [inline recipes/libnabo/all/patches/1.1.0-0001-fix-cmake.patch:1]: The fPIC is forced to ON always when building static. Could you please remove it, so Conan could manage it according to self.options.fPIC instead?

https://github.com/norlab-ulaval/libnabo/compare/1.0.7...1.1.0#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR149

Comment (toge) [reply recipes/libnabo/all/patches/1.1.0-0001-fix-cmake.patch:1]: @uilianries 
Thanks!
I will apply your point.

Comment (toge) [reply recipes/libnabo/all/patches/1.1.0-0001-fix-cmake.patch:1]: @uilianries 
Fixed.
Please review again.

**Review by uilianries — APPROVED** [review]
LGTM

**Review by RubenRBS — APPROVED** [review]
Thanks!

**Pull request merged**